map_view/show-location-history/README.metadata.json

```
@@ -0,0 +1,45 @@
+{
+    "category": "Map view",
+    "description": "Display your location history on the map.",
+    "ignore": false,
+    "images": [
+        "ShowLocationHistory.jpg"
```

This should say 
```suggestion
        "ShowLocationHistory.png"
```

If you haven't been using the metadata script to generate this file, it's worth giving it a try, since it does deal with quite a lot of the things that may have typos in it!